docs/kpod-rm.1.md

```diff
@@ -17,13 +17,18 @@ Kpod rm will remove one or more containers from the host.  The container name or
 
 Force the removal of a running container
 
+**--all, a**
+
+Remove all containers.  Can be used in conjunction with -f as well.
 
 ## EXAMPLE
 
 kpod rm mywebserver
 
 kpod rm -f 860a4b23
 
+kpod rm -f -a
```

Comment:
I know it's not your change, but it would be nice to show how to do multiple container removes as an example in addition to the single one here.  Ditto at line 12 it starts with "Kpod run", it  should be "kpod run"

Reply:
fixed both